### ISSUE TITLE
Isn't this supposed to be dict()

### ISSUE THREAD

**Issue by NotSwaraj** [Nyumat/The-TikTok-Bot#19]
https://github.com/Nyumat/The-TikTok-Bot/blob/48a3b1b3b9df9b4f6b4cb1fb1f692fd3f2029bc9/bot/requests-V1/main.py#L152 hmmm

**Comment by Nyumat** [comment]
> https://github.com/Nyumat/The-TikTok-Bot/blob/48a3b1b3b9df9b4f6b4cb1fb1f692fd3f2029bc9/bot/requests-V1/main.py#L152 hmmm 

Yeah, but if you read the comments you'd know this file is out of production...10 months ago

**Comment by NotSwaraj** [comment]
Alright.. Sorry to bother you!
